client/src/components/TypeEditor.vue

```diff
@@ -3,13 +3,12 @@ import {
   defineComponent, PropType, reactive, toRef, watch,
 } from '@vue/composition-api';
 
-import BaseFilterControls from 'vue-media-annotator/BaseFilterControls';
-import Group from 'vue-media-annotator/Group';
-import StyleManager from 'vue-media-annotator/StyleManager';
-import Track from 'vue-media-annotator/track';
-
 import { usePrompt } from 'dive-common/vue-utilities/prompt-service';
 
+import BaseFilterControls from '../BaseFilterControls';
+import type Group from '../Group';
+import type StyleManager from '../StyleManager';
+import type Track from '../track';
 import { useReadOnlyMode } from '../provides';
 
 export default defineComponent({
```

Comment:
I still think this component needs some props to reference properly the differences between track data and group data.

Stuff like the confidence/label checkboxes don't work properly.
The concept of deleting a type doesn't make sense because there are no empty group types that can be deleted.

Reply:
I meant the concept of having the Delete Type button in the `TypeEditor.vue` for Group Types (I know confusing).  I don't think this button will ever become active because there is no cause in which there will be 0 groups associated with a group type.  Where as with track types we can have empty types.  There is no way to add empty group types (I didn't see one yet).
![image](https://user-images.githubusercontent.com/61746913/168826642-80148a02-22bb-4a71-8e5b-e77255bea56e.png)